contracts/osmosis-vault/Cargo.toml

```diff
@@ -18,10 +18,12 @@ exclude = [
 crate-type = ["cdylib", "rlib"]
 
 [features]
+default = ["osmosis-test-tube"]
 # for more explicit tests, cargo test --features=backtraces
 backtraces = ["cosmwasm-std/backtraces"]
 # use library feature to disable all instantiate/execute/query exports
 library = []
+osmosis-test-tube = ["cw-it/osmosis-test-tube", "apollo-vault-test-helpers/osmosis-test-tube"]
```

Comment:
why do we need this? can we not just import these crates with the osmosis-test-tube feature by default?

Reply:
The only reason is because debugging does not work when `cw-it/osmosis-test-tube` is enabled.